src/checks_windows.go

```diff
@@ -204,10 +216,52 @@ func packageInstalled(packageName string) (bool, error) {
 }
 
 func serviceUp(serviceName string) (bool, error) {
-	return commandOutput("(Get-Service -Name '"+serviceName+"').Status", "Running")
+	serviceStatus, err := getLocalServiceStatus(serviceName)
+	return serviceStatus.IsRunning, err
+}
+
+func serviceStatus(serviceName, wantedStatus, startupType string) (bool, error) {
+	status, err := getLocalServiceStatus(serviceName)
+	var boolWantedStatus bool
+	if err != nil {
+		return false, err
+	}
+	wantedStatus = strings.ToLower(wantedStatus)
+	if wantedStatus == "running" {
+		boolWantedStatus = true
+	} else if wantedStatus == "stopped" {
+		boolWantedStatus = false
+	} else {
+		errMessage := `Unknown startup type found for `+serviceName
```

Comment:
spacing, extra use of `

Reply:
wat

Reply:
errMessage := "Unknown startup type found for " + serviceName